morse_code.rb

```diff
@@ -0,0 +1,26 @@
+def decode_char(char)
+  morse_code = {
+    '.-' => 'A', '-...' => 'B',
+    '-.-.' => 'C', '-..' => 'D',
+    '.' => 'E', '..-.' => 'F',
+    '--.' => 'G', '....' => 'H',
+    '..' => 'I', '.---' => 'J',
+    '-.-' => 'K', '.-..' => 'L',
+    '--' => 'M', '-.' => 'N',
+    '---' => 'O', '.--.' => 'P',
+    '--.-' => 'Q', '.-.' => 'R',
+    '...' => 'S', '-' => 'T',
+    '..-' => 'U', '...-' => 'V',
+    '.--' => 'W', '-..-' => 'X',
+    '-.--' => 'Y', '--..' => 'Z'
+  }
+  morse_code.fetch(char, '')
+end
+
+def decode_word(word)
+  word.split.map { |char| decode_char(char) }.join
+end
+
+def decode(message)
+  message.split('   ').map { |word| decode_word(word) }.join(' ')
+end
```

Comment:
- [ ] Kinldy make sure you provide how to test your code before submission. which method is your final? Is `decode_word`, `decode`, and also what the `decode_char` is for?

Please make sure you test your algorithm like :
```rb
decode(morse_code)
```